XcodeServerSDK/XcodeServer.swift

```diff
@@ -378,9 +410,154 @@ public extension XcodeServer {
         }
     }
     
-//    public func reportQueueSizeAndEstimatedWaitingTime(integration: Integration, completion: ((queueSize: Int, estWait: Double), NSError?) -> ()) {
+    public func getPlatforms(completion: (platforms: [Platform]?, error: NSError?) -> ()) {
```

Comment:
Why they need to specify a `completion` block?

From the look of the code everything is synchronous; wouldn't it be more readable like:
```swift
public func getPlatforms() throws -> [Platform] {
```

That way we also get rid of the need to check if the returned array is `nil` or not and if there's an error plus the `nil` check.